### DISCUSSION TITLE
Chat hotkeys not working

### DISCUSSION THREAD

**Discussion by GreatNashira** [SnosMe/awakened-poe-trade#745] [Q&A]
When I press F5 or F9 nothing happens. The program was working fine for months and today it stopped and does not work. I have tried running it in admin and not admin. When I press F5 and check the chat log it is sending the /hideout command but nothing happens. I checked and the press enter option is enabled but it is not working. IF I run POE in admin then the hotkeys work but I would prefer not to have to do that.

**Comment by SnosMe** [comment]
No changes were made to that part of code. You need to run tool elevated after Steam self-updates, because Steam continues to run after that in elevated mode and launches all games with admin rights too.

**Reply by GreatNashira** [reply]
I am running it elevated. What's strange is everything else works. If I turn off the "hit enter" option it will input the command into chat and I just have to hit enter, so that is the part that is not working. This only other thing I did on my end was install auto hot key which I have since uninstalled. I also know that steam is NOT running as an admin as if I force it to run admin the problem is also resolved. If there are troubleshooting steps I can take to get you more info on this I would like to do so as this is a very bizarre problem.

**Comment by AuntDoozie** [comment]
I know this is a few years old now but I am running into this EXACT same problem. Running Awakened Trade as admin does not work, I am not launching poe from steam, but launching poe with admin rights works for some reason.
